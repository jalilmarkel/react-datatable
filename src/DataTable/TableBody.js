import styled from 'styled-components';

const TableBody = styled.div`
  /* future styling */
`;

export default TableBody;
