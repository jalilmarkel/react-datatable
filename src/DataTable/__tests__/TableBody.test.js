import 'jest-styled-components';
import React from 'react';
import { render, cleanup } from 'react-testing-library';
import TableBody from '../TableBody';

afterEach(cleanup);

test('component <TableBody /> should render correctly', () => {
  const { container } = render(<TableBody />);
  expect(container.firstChild).toMatchSnapshot();
});
