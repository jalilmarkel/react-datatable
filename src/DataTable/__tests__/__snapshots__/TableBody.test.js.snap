// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`component <TableBody /> should render correctly 1`] = `
<div
  class=""
/>
`;
